### ISSUE TITLE
Error summary panel heading isn't marked up as a heading.  (probably should be an `h1`) See: https://github.com/alphagov/govuk_elements/pull/510

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#1079]
Error summary panel heading isn't marked up as a heading.  (probably should be an `h1`) See: https://github.com/alphagov/govuk_elements/pull/510

_Originally posted by @jrbarnes9 in https://github.com/ONSdigital/design-system/issue_comments/706046488_
